content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -53,3 +64,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+
+Once you are satisfied with the Payslips, click :guilabel:`Create Draft entry` to generate a draft
+journal entry found in the :guilabel:`Other Info` tab of each payslips.
+
+Individual payslips
+~~~~~~~~~~~~~~~~~~~
+
+Go to :menuselection:`Payroll --> Payslips --> All Payslips`
+This method of payslip generation is commonly used to handle one-off payments (e.g. Payment in Lieu
+of Notice, Long Service Payment, Severance Payment).
+
+#. Click on :guilabel:`New`.
+#. Select an :guilabel:`Employee`; their :guilabel:`Contract` will be filled out automatically
+#. Add a pay :guilabel:`Period`
+#. Select a salary :guilabel:`Structure` (e.g. Employees Monthly Pay)
+#. The **Worked Days** tab will automatically compute the worked days/hours and time-off leaves
+   that are applicable.
+#. You may also choose to add additional payslip items (e.g. Commissions, Deductions) under the
+   **Other Inputs** section.
+#. Click on :guilabel:`Compute Sheet` button to generate the payslip lines. This button will update
+   the :guilabel:`Salary Computation` tab.
+
+.. image:: hong_kong/hk-individual-payslip.png
+   :alt: Hong Kong Individual Payslip
+
+.. note::
+   If you have amended the work entry for an employee, you will need to go to the **gear icon (⚙)**
+   and click on **Recompute Whole Sheet** to refresh the payslip's **Worked Day & Inputs** section.
+
+The :guilabel:`Salary Computation` tab will show the detailed breakdown of the computation based on
+the salary rules configured for each structure type.
+
+.. image:: hong_kong/hk-salary-computation.png
+   :alt: Hong Kong Salary computation
+
+#. **Rent Allowance**: Amount derived from the employee's active rental record.
+#. **Basic salary**: Amount from which tax-deductible lines can be subtracted minus rental
+   allowance.
+#. **713 Gross**: Amount after considering allowances/deductions.
+#. **MPF Gross**
+#. **Employee Mandatory Contribution**: Employee MPF Contribution
+#. **Employer Mandatory Contribution**: Employer MPF Contribution
+#. **Gross**
+#. **Net Salary**: Final amount to be paid to the employee
+#. **Monthly End Autopay**
+#. **Other Input Types**: Depends on whether there are additional items to be included in the payslip.
+
+.. note::
+   Do note there will be no MPF contribution for the first month. Both **employee** and **employer**
+   contribution starts on second month and the **employer** contribution for the second month will
+   include back-pay of the first month.
+
+Under the :guilabel:`Other Inputs` tab at the bottom of payslip, there are additional manual input
+types that are specific to HK Payroll:
+
+- :guilabel:`Back Pay`
+- :guilabel:`Commission`
+- :guilabel:`Global Deduction`
+- :guilabel:`Global Reimbursement`
+- :guilabel:`Referral Fee`
+- :guilabel:`Moving Daily Wage`: if you wish to override the Average Daily Wage (ADW) for this month.
+- :guilabel:`Skip Rent Allowance`
+- :guilabel:`Custom Average Monthly Salary`: only affecting end-of-year payments.
```

Comment:
these should all be defined- there's only a few here, but to be consistent, everything should have an explanation, even if it sounds obvious.